modules/claims_api/spec/requests/v2/veterans/rswag_disability_compensation_request_spec.rb

```diff
@@ -61,10 +61,18 @@
         response '202', 'Successful response with disability' do
           schema SwaggerSharedComponents::V2.schemas[:disability_compensation]
           let(:scopes) { %w[system/claim.read system/claim.write] }
+          let(:claim_date) { (Time.zone.today - 1.day).to_s }
```

Comment:
When running the test suite this test also fails.  I think this line has to go and the one below on line 71.